plugins/modules/clickhouse_user.py

```diff
@@ -75,6 +75,27 @@
     type: str
     choices: [always, on_create]
     default: on_create
+  user_hosts:
+    description:
+      - Host restrictions to apply to the user.
+      - It's a list of dictionaries, where each dictionary specifies the type of restriction to apply to which hosts or pattern.
+    type: list
+    elements: dict
+    suboptions:
+      type:
+        description:
+          - The method used to validate which hosts that users are allowed to connect from (IP, ANY, LOCAL, NAME, REGEXP...).
```

Comment:
```suggestion
          - The method used to validate which hosts that users are allowed to connect from (V(IP), V(ANY), V(LOCAL), V(NAME), V(REGEXP)...).
```
also what do `...` mean? Sorry, I'm not a user.